### DISCUSSION TITLE
Feature Request: Build a smaller docker image without gpu support

### DISCUSSION THREAD

**Discussion by Zxilly** [open-webui/open-webui#5141] [General]
**Is your feature request related to a problem? Please describe.**

Currently only unique all-in-one image is built for ollama, and if the user doesn't use the gpu, a large portion of that binary volume is meaningless.

**Describe the solution you'd like**

Release a smaller image that does not include GPU support, allowing users to pick and choose the features they need.

**Describe alternatives you've considered**

I don't think there is any other way to avoid it, because of the image layer, the user can't reduce the size of the image at runtime.

**Comment by rwjack** [comment]
I don't get this either, I am using the GPU, but why TF is the image 8.5GB??

Looking at https://github.com/open-webui/open-webui/discussions/5820:

> https://github.com/ollama-webui/ollama-webui-lite would be a good alternative in case anyone's looking for just the "UI", which Open WebUI is not.

ollama-webui-lite seems to be unmaintained. 

So can someone please clarify what does Open WebUI do then? Isn't all the brainpower in the ollama backend? The UI in the name has me confused.
